src/storage/mvcc/reader/backward_scanner.rs

```diff
@@ -167,8 +168,16 @@ impl<S: Snapshot> BackwardScanner<S> {
     /// Get the next key-value pair, in backward order.
     pub fn read_next(&mut self) -> Result<Option<(Key, Value)>> {
         if !self.is_started {
-            self.write_cursor.seek_to_last(&mut self.statistics.write);
-            self.lock_cursor.seek_to_last(&mut self.statistics.lock);
+            // TODO: `seek_to_last` is better, however it has performance issues currently.
+            // TODO: We can eliminate clones here.
+            self.write_cursor.reverse_seek(
```

Comment:
Yes. However there is 100% reproduceable notable performance difference. Since this PR is going to be merged very soon, we should keep it first. After that, we can investigate what's wrong with seek_to_last.